package/src/RNCSliderNativeComponent.ts

```diff
@@ -41,6 +41,6 @@ export interface NativeProps extends ViewProps {
   upperLimit?: Float;
 }
 
-export default codegenNativeComponent<NativeProps>(
-  'RNCSlider',
-) as HostComponent<NativeProps>;
+export default codegenNativeComponent<NativeProps>('RNCSlider', {
+  interfaceOnly: true,
```

Comment:
Q: What's the reason you don't use the generate C++ code and expand from it?

Reply:
So when I removed this I got a new issue saying that I'm redefining ShadowNode with different types - how can I achieve what you mentioned to expand from generated C++ code? 

![image](https://github.com/callstack/react-native-slider/assets/52801365/3dec55ad-2ebe-488e-8c05-c00c9245515b)

Reply:
Ok so it's a bit more complicated. Are you fine with merging this as it is, as it unblocks New Architecture migration, and then we can follow-up on a cleanup afterwards?